lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1263,7 +1263,7 @@ protected function get_layout_styles( $block_metadata ) {
 
 		if ( isset( $block_metadata['name'] ) ) {
 			$block_type = WP_Block_Type_Registry::get_instance()->get_registered( $block_metadata['name'] );
-			if ( ! block_has_support( $block_type, array( '__experimentalLayout' ), false ) ) {
+			if ( ! block_has_support( $block_type, array( 'layout' ), false ) ) {
```

Comment:
Should this one also fall back for `__experimentalLayout` if `layout` isn't available?

Reply:
Might as well add it in!